.travis.yml

```diff
@@ -104,22 +104,22 @@ matrix:
     #- os: osx
     #  osx_image: xcode6.4
     #  compiler: clang
-    #  env: COMPILER='clang++' BUILD_TYPE='Debug'
+    #  env: CCOMPILER='clang' CXXCOMPILER='clang++' BUILD_TYPE='Debug'
 
     #- os: osx
     #  osx_image: xcode6.4
     #  compiler: clang
-    #  env: COMPILER='clang++' BUILD_TYPE='Release'
+    #  env: CCOMPILER='clang' CXXCOMPILER='clang++' BUILD_TYPE='Release'
 
     #- os: osx
     #  osx_image: xcode6.4
     #  compiler: clang
-    #  env: COMPILER='clang++' BUILD_TYPE='Release' BUILD_SHARED_LIBS=ON
+    #  env: CCOMPILER='clang' CXXCOMPILER='clang++' BUILD_TYPE='Release' BUILD_SHARED_LIBS=ON
 
     #- os: osx
     #  osx_image: xcode7
     #  compiler: clang
-    #  env: COMPILER='clang++' BUILD_TYPE='Release' BUILD_SHARED_LIBS=ON
+    #  env: CCOMPILER='clang' CXXCOMPILER='clang++' BUILD_TYPE='Release' BUILD_SHARED_LIBS=ON
```

Comment:
We disabled shared library builds on OSX because 1/ nobody uses that and it increases build time and 2/ it had some weird linking issues that nobody wanted to debug.